IPs/RSKIP428.md

```diff
@@ -0,0 +1,75 @@
+---
+rskip: 428
+title: Emit a new event with the utxo outpoints when a peg-out is created
+created: 23-APR-24
+author: NC
+purpose: Sca
+layer: Core 
+complexity: 1
+status: Draft
+description: 
+---
+
+|RSKIP          | 428                                                                |
+| :------------ |:-------------------------------------------------------------------|
+|**Title**      | Emit a new event with the utxo outpoints when a peg-out is created |
+|**Created**    | 23-APR-24                                                          |
+|**Author**     | NC                                                                 |
+|**Purpose**    | Sca                                                                |
+|**Layer**      | Core                                                               |
+|**Complexity** | 1                                                                  |
+|**Status**     | Draft                                                              |
+
+## Abstract
+
+This RSKIP proposes creating a new event that stores the outpoint values of the inputs used in a peg-out to enable the PowHSM sign segwit peg-outs.
+
+## Motivation
+
+For the PowHSM sign segwit peg-outs[1], the outpoint values of the inputs must be provided since they are needed for calculating the sighash of a segwit bitcoin transaction. Currently, selected UTXOs for a peg-out are removed from the available utxos set once the peg-out is created, meaning they cannot be retrieved later. Storing the outpoint values in an event, when peg-outs are created and before the selected utxos are removed from the set, is a solution that will allow the PowPeg to retrieve and provide the required outpoint values to the PowHSM to sign segwit peg-outs.
+
+## Specification
+
+The proposal is to create a new event to store the bitcoin transaction hash and the UTXO outpoint values of a transaction spending funds from the federation, which from now on we will call it peg-out. This event will be emitted when a peg-out is created.
```

Comment:
```suggestion
The proposal is to create a new event to store the bitcoin transaction hash and the UTXO outpoint values of a transaction spending funds from the PowPeg address. This could be either a peg-out, a migration, or a refund transaction. The event will be emitted whenever any of these transactions are created.
```